deploy/README.md

```diff
@@ -0,0 +1,7 @@
+# How to create a validation CRD for mpijob
+
+* Submit a PR to update the vendor version for [crd-validation](https://github.com/github.com/crd-validation)
+* Build and generate a new API validation CRD
+  * build the [crd-validation](https://github.com/github.com/crd-validation)
+  * `./crd-validation -c ./examples/mpijob/crd-validation.yaml mpijob`
+* Copy it to this directory
```

Comment:
Perhaps add this as a script in /hack or Makefile? We also need to include a git hash for the crd-validation that we used

Reply:
sure.